src/components/StaticPages/About.js

```diff
@@ -14,6 +16,34 @@ export default () => {
   return (
     <div>
```

Comment:
Should add a CSS classname here and put all above css styles into the classname.